addons/mail/wizard/base_module_install_request.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class BaseModuleInstallRequest(models.TransientModel):
+    _name = "base.module.install.request"
```

Comment:
You should probably have a small module for that to avoid bloating mail with unrelated content.